one_fm/operations/doctype/process_task/process_task.py

```diff
@@ -427,8 +427,8 @@ def run_cron_based_process_tasks():
 def run_cron_process_task():
 	"""Trigger all the Task creating process tasks for the day for cron based process tasks"""
 	try:
-		time_now = datetime.now()
-		today = datetime.today()
+		time_now = frappe.utils.now_datetime()
+		today = frappe.utils.get_datetime()
```

Comment:
We imported get_datetime and now_datetime

https://github.com/ONE-F-M/one_fm/blob/cc40697e0a7e2a5980929f96b13172d3d78ac186/one_fm/operations/doctype/process_task/process_task.py#L17

https://github.com/ONE-F-M/one_fm/blob/cc40697e0a7e2a5980929f96b13172d3d78ac186/one_fm/operations/doctype/process_task/process_task.py#L15

Reply:
updated @pjamsheer